README.md

```diff
@@ -185,10 +161,10 @@ An item required in exchange for an offer.
 ```protobuf
 message ConsiderationItem {
   ItemType item_type = 1;
-  H160 token = 2; // address
-  H256 identifier_or_criteria = 3; // uint256
-  H256 start_amount = 4; // uint256
-  H256 end_amount = 5; // uint256
```

Comment:
Personally I found these really helpful during my onboarding.

Would it be reasonable to add the comparable Solidity types to the type headers?
ie:
```
### H160 

A 160-bit data type. Used to store 20-byte addresses.
[...]
### H256

A 256-bit data type. Equivalent to solidity's uint256, bytes32string, etc.
```

Something like that maybe? Just a thought